nodes/logic/neuro_elman.py

```diff
@@ -197,70 +202,68 @@ def sv_init(self, context):
-        layout.prop(self, "gisterezis", text="gisterezis")
+
+        col_top = layout.column(align=True)
+        row = col_top.row(align=True)
+        row.prop(self, "lA", text="A layer")
+        row = col_top.row(align=True)
+        row.prop(self, "lB", text="B layer")
+        row = col_top.row(align=True)
+        row.prop(self, "lC", text="C layer")
+
         layout.prop(self, "maximum", text="maximum")
-        layout.prop(self, "cycles", text="cycles")
-        op_start = layout.operator('node.sverchok_neuro', text='Restart')
-        op_start.typ=1
+        op_start = layout.operator('node.sverchok_neuro', text='Reset')
+        op_start.typ = 1
         op_start.handle_name = handle_name
         layout.prop(self, "menushka", text="extend sets:")
         if self.menushka:
-            col_top = layout.column(align=True)
-            row = col_top.row(align=True)
-            row.prop(self, "lA", text="A layer")
-            row = col_top.row(align=True)
-            row.prop(self, "lB", text="B layer")
-            row = col_top.row(align=True)
-            row.prop(self, "lC", text="C layer")
+            layout.prop(self, "k_learning", text="koeff learning")
+            layout.prop(self, "gisterezis", text="gisterezis")
+            layout.prop(self, "cycles", text="cycles")
             col = layout.column(align=True)
             col.prop(self, "epsilon", text="epsilon")
             col = layout.column(align=True)
             col.prop(self, "k_lambda", text="lambda")
             col = layout.column(align=True)
             col.prop(self, "treshold", text="treshold")
 
-
     def process(self):
         handle_name = self.name + self.id_data.name
         handle = handle_read(handle_name)
         props = handle[1]
         if not handle[0]:
-            props = {'InA':2,
-                     'InB':5,
-                     'InC':1,
-                     'wA':[],
-                     'wB':[],
-                     'gister':0.01,
-                     'k_learning':0.1,
-                     'epsilon':1.3,
-                     'cycles':3,
-                     'trashold':0.01,
-                     'k_lambda':0.0001}
-
-            props['wA'] = self.Elman.init_w(props['InA'], props['InB'], props['trashold'])
-            props['wB'] = self.Elman.init_w(props['InB'], props['InC'], props['trashold'])
-
-
-        self.Elman.gister = abs(self.gisterezis)
-        self.Elman.k_learning = self.k_learning
-
+            elman = SvNeuroElman()
+            props = {'InA': 2,
+                     'InB': 5,
+                     'InC': 1,
+                     'wA': [],
+                     'wB': [],
+                     'gister': 0.01,
+                     'k_learning': 0.1,
+                     'epsilon': 1.3,
+                     'cycles': 3,
+                     'trashold': 0.01,
+                     'k_lambda': 0.0001,
+                     'Elman': elman,
+                     }
```

Comment:
Probably it would be good to use data class here because it will let use dot notation.
https://docs.python.org/3.9/library/dataclasses.html